lib/ansible/modules/windows/win_domain_controller.ps1

```diff
@@ -24,6 +22,9 @@ Function Write-DebugLog {
 
     Write-Debug $msg
 
+    $log_path = $null
```

Comment:
Not sure why this is also set, you are setting the var again below.

Reply:
that's powershell best practice to init vars before assigning to them

Reply:
There's no need, you are immediately assigning that var to something else just below it. If it was conditionally defined then I would understand but in this case it's literally the next line.

Reply:
since the assignment is from a function it is ill preferred to initialize the var to a known value